tests-bundle/1.7/test_1dot7.py

```diff
@@ -41,7 +41,7 @@ async def test_deploy(ops_test: OpsTest, lightkube_client, deploy_cmd):
         'kfp-viz',
         # 'knative-eventing', # this is expected to wait for config
         'knative-operator',
-        'knative-serving',  # this is expected to wait for config
+        # 'knative-serving', # this is expected to wait for config
```

Comment:
Why is it commented out?

Reply:
there are some apps that cause troubles in CI before we run config in DEX in OIDC